tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -0,0 +1,333 @@
+&atmos_model_nml
+    blocksize = 32
+    ccpp_suite = '@[CCPP_SUITE]'
+    chksum_debug = .false.
+    dycore_only = .false.
+/
+
+&cires_ugwp_nml
+    knob_ugwp_azdir = 2, 4, 4, 4
+    knob_ugwp_doaxyz = 1
+    knob_ugwp_doheat = 1
+    knob_ugwp_dokdis = 1
+    knob_ugwp_effac = 1, 1, 1, 1
+    knob_ugwp_ndx4lh = 1
+    knob_ugwp_solver = 2
+    knob_ugwp_source = 1, 1, 0, 0
+    knob_ugwp_stoch = 0, 0, 0, 0
+    knob_ugwp_version = 0
+    knob_ugwp_wvspec = 1, 25, 25, 25
+    launch_level = 25
+/
+
+&diag_manager_nml
+    prepend_date = .false.
+    max_output_fields = @[MAX_OUTPUT_FIELDS]
+/
+
+&external_ic_nml
+    checker_tr = .false.
+    filtered_terrain = .true.
+    gfs_dwinds = .true.
+    levp = 66
+    nt_checker = 0
+/
+
+&fms_io_nml
+    checksum_required = .false.
+    max_files_r = 100
+    max_files_w = 100
+/
+
+&fms_nml
+    clock_grain = 'ROUTINE'
+    domains_stack_size = 3000000
+    print_memory_usage = .false.
+/
+
+&fv_core_nml
+    a_imp = 1.0
+    adjust_dry_mass = .false.
+    bc_update_interval = 1
+    beta = 0.0
+    consv_am = .false.
+    consv_te = 0.0
+    d2_bg = 0.0
+    d2_bg_k1 = 0.2
+    d2_bg_k2 = 0
+    d4_bg = 0.15
+    d_con = 0.5
+    d_ext = 0.0
+    dddmp = 0.1
+    delt_max = 0.008
+    dnats = @[DNATS]
+    do_sat_adj = @[DO_SAT_ADJ]
+    do_schmidt = .true.
+    do_vort_damp = .true.
+    dwind_2d = .false.
+    dz_min = 6.0
+    external_eta = .true.
+    external_ic = @[EXTERNAL_IC]
+    fill = .true.
+    full_zs_filter = .false.
+    fv_debug = .false.
+    fv_sg_adj = 300
+    gfs_phil = .false.
+    hord_dp = -5
+    hord_mt = 5
+    hord_tm = 5
+    hord_tr = 10
+    hord_vt = 5
+    hydrostatic = .false.
+    io_layout = 1, 1
+    k_split = 6
+    ke_bg = 0.0
+    kord_mt = 9
+    kord_tm = -9
+    kord_tr = 9
+    kord_wz = 9
+    layout = @[INPES],@[JNPES]
+    make_nh = @[MAKE_NH]
+    mountain = @[MOUNTAIN]
+    n_split = 3
+    n_sponge = 9
+    n_zs_filter = 0
+    na_init = @[NA_INIT]
+    ncep_ic = .false.
+    nggps_ic = @[NGGPS_IC]
+    no_dycore = .false.
+    nord = 2
+    nord_tr = 2
+    npx = @[NPX]
+    npy = @[NPY]
+    npz = @[NPZ]
+    nrows_blend = 20
+    ntiles = 1
+    nudge_qv = .false.
+    nwat = 6
+    p_fac = 0.1
+    phys_hydrostatic = .false.
+    print_freq = 6
+    psm_bc = 1
+    range_warn = .true.
+    read_increment = @[READ_INCREMENT]
+    regional = .true.
+    regional_bcs_from_gsi = .false.
+    res_latlon_dynamics = @[RES_LATLON_DYNAMICS]
+    reset_eta = .false.
+    rf_cutoff = 20000.0
+    sg_cutoff = 10000.0
+    stretch_fac = 0.999
+    target_lat = 38.5
+    target_lon = -97.5
+    tau = 5.0
+    use_hydro_pressure = .false.
+    vtdm4 = 0.03
+    warm_start = @[WARM_START]
+    write_restart_with_bcs = .false.
+    z_tracer = .true.
+/
+
+&fv_grid_nml
+    grid_file = 'INPUT/grid_spec.nc'
+/
+
+&gfs_physics_nml
+    @[FH_DFI_RADAR]
```

Comment:
The syntax for the new `FH_DFI_RADAR` is different from all the other entries, which are of the form
```
    cdmbgwd = @[CDMBWD]
```
Why do we need to add more confusion? Can't we have something like
```
fh_dfi_radar = @[FH_DFI_RADAR]
```
with the default values for `FH_DFI_RADAR` in `default_vars.sh` set so that the radar tendencies are turned off unless the user specifies something else?

Reply:
Turning off the feature requires not specifying fh_dfi_radar.

Reply:
@SamuelTrahanNOAA I see inside fv3, the default values are set to -2e10 when the scheme is not call. Please use this default values. It will be consistent with other namelist variable settings. Otherwise it's confusing and error prone. Thanks

Reply:
Test passed. Changes are merged.